core/pkg/opencost/allocation.go

```diff
@@ -100,9 +100,64 @@ type Allocation struct {
 	// UnmountedPVCost is used to track how much of the cost in PVs is for an
 	// unmounted PV. It is not additive of PVCost() and need not be sent in API
 	// responses.
-	UnmountedPVCost   float64 `json:"-"`                 //@bingen:field[ignore]
-	GPURequestAverage float64 `json:"gpuRequestAverage"` //@bingen:field[version=22]
-	GPUUsageAverage   float64 `json:"gpuUsageAverage"`   //@bingen:field[version=22]
+	UnmountedPVCost             float64        `json:"-"`                 //@bingen:field[ignore]
+	DeprecatedGPURequestAverage float64        `json:"gpuRequestAverage"` //@bingen:field[version=22]
+	DeprecatedGPUUsageAverage   float64        `json:"gpuUsageAverage"`   //@bingen:field[version=22]
```

Comment:
I would change these fields to the following:

```go
deprecatedGPURequestAverage float64        `json:"-"` //@bingen:field[version=22]
deprecatedGPUUsageAverage   float64        `json:"-"`   //@bingen:field[version=22]
```

We don't want these fields to be exposed at all outside of the package (since they're now deprecated). Also, if at all possible, we don't want these fields to appear in the JSON encoding either.

Reply:
You can just rename these and re-run bingen.

Reply:
Done. Re-running bingen didnt do anything in `bingen_codecs.go`, is that expected?